org/mozilla/jss/tests/TestSSLEngine.java

```diff
@@ -66,13 +77,16 @@ public static void testBasics(SSLContext ctx) throws Exception {
 
         // Tests {get,set}EnabledProtocols()
         String[] protocols = ssle.getSupportedProtocols();
-        assert protocols.length >= 2;
-        String secondProtocol = protocols[1];
-        String[] oneProtocols = new String[]{ secondProtocol };
+
+        // Fedora returns at least 2 supported protocols
+        // in FIPS mode, but RHEL returns only 1.
+        assert protocols.length >= 1;
+        String firstProtocol = protocols[1];
```

Comment:
this still is the `secondProtocol` right?

Reply:
Fixed. It should be the first protocol.